app/src/components/blocks/selectStage/SelectStage.tsx

```diff
@@ -28,7 +28,9 @@ function SelectStage({ uploadDocuments, setDocuments, patientDetails }: Props) {
 
     const mergedDocuments = [...arfDocuments, ...lgDocuments];
     const hasFileInput = mergedDocuments.length;
-
+    const lgRegex =
+        /[0-9]+of[0-9]+_Lloyd_George_Record_\[[A-Za-z]+\s[A-Za-z]+]_\[[0-9]{10}]_\[\d\d-\d\d-\d\d\d\d].pdf/; // eslint-disable-line
```

Comment:
Do we want an unlimited amount of characters for forename and surname?

Reply:
No limit for now